eclass/git-r3.eclass

```diff
@@ -682,6 +682,8 @@ git-r3_fetch() {
 					# and HEAD in case we need the default branch
 					# (we keep it in refs/git-r3 since otherwise --prune interferes)
 					"+HEAD:refs/git-r3/HEAD"
+					# fetch the specifc commit_ref to deal with orphan commits
+					"${remote_ref}"
```

Comment:
Is it actually safe to do it like this? Perhaps the `fetch_l` logic from below should be moved upwards, and reused here.

Reply:
It should be safe as fetching the specific reference would at most be redundant if its not orphaned. I don't see the mirror logic getting value from the fetch_l logic as it this is just for the edge case of orphan commits.

Reply:
Right, I don't see how it could be unsafe, and given there's two broken live ebuilds rn, merged.